contracts/p1/BasketHandler.sol

```diff
@@ -339,19 +340,21 @@ contract BasketHandlerP1 is ComponentP1, IBasketHandler {
             (block.timestamp >= lastStatusTimestamp + warmupPeriod);
     }
 
+    /// Basket quantity rounded up, without any issuance premium
     /// @param erc20 The token contract to check for quantity for
     /// @return {tok/BU} The token-quantity of an ERC20 token in the basket.
     // Returns 0 if erc20 is not registered or not in the basket
     // Returns FIX_MAX (in lieu of +infinity) if Collateral.refPerTok() is 0.
     // Otherwise returns (token's basket.refAmts / token's Collateral.refPerTok())
     function quantity(IERC20 erc20) public view returns (uint192) {
```

Comment:
Noting that all external functions now do not expose the premium, so maybe worth adding another getter here to allow passing issuancePremium flag.

Reply:
Not quite -- `quote(, CEIL)` and `price()` expose the premium. Agreed no `quantity*()` functions do though. I don't think we need the flag.

Reply:
The BackingManager has a call to `quantityUnsafe()` that _shouldn't_ use the flag. If it makes sense to add the flag I think it's because something outside our system needs it. So the question is whether `quote()` is sufficient.

Reply:
I think it's worth adding another function, purely because an external system might need to know the premium on the actual basket instead of just a quote even though both can be derived from each other.

Reply:
There isn't a premium at the basket level, conceptually. Individual collateral can be required more of, but you can't combine the quantities of the different tokens without something like applying a USD lens. So it doesn't make sense to me to expose premium at a basket level. 

`quote()` seems to offload _more_ work for integrators here, no? You can directly divide the quantities for the `quote(FLOOR)` and `quote(CEIL)` baskets to see how much premium is being applied.

Reply:
Trying to steelman...I guess is someone is writing an on-chain integration that only cares about one collateral, then they'd want individual collateral level access? Mmm I think I see what to do here...

Reply:
Trying to steelman...I guess is someone is writing an on-chain integration that only cares about one collateral, then they'd want individual collateral level access? Mmm I think I see what to do here...

Reply:
ok, added a getter for what I think they would care about. just under the contract size!